### DISCUSSION TITLE
误打误撞

### DISCUSSION THREAD

**Discussion by sfhjhsdfg** [share121/inter-knot#1264] [其他]
误打误撞注册了这个网站，作为一个在十一分街生活的良好市民，绳网内的内容对我大受震撼，这不会是个灰色网站吧，我还看到这个网站里有人在发委托

**Comment by gaojiaobei** [comment]
是的，我发了一个帖子，结果被治安官抓住枪毙了2小时
还好我是体育生，休息一会就好了
